src/lib/netlist/plib/plists.h

```diff
@@ -39,7 +39,7 @@ namespace plib {
 		using reverse_iterator = std::reverse_iterator<iterator>;
 		using const_reverse_iterator = std::reverse_iterator<const_iterator>;
 
-		//uninitialised_array_t() noexcept = default;
+		//uninitialized_array_t() noexcept = default;
 		uninitialised_array() noexcept = default;
```

Comment:
What’s the actual purpose of the comment here?  The comment uses US spelling while the code uses Commonwealth spelling for the class name.  It looks like a leftover from some kind of refactoring.

Reply:
> It looks like a leftover from some kind of refactoring.

Yes, just delete the comment. What spelling is preferred? cspell should support both en-US and en-GB.

Reply:
> What spelling is preferred?

I don’t care too much as long as it’s consistent across a unit.  Executive summaries for release announcements are written in Commonwealth English, but the summarised pull request descriptions preserve the author’s preference for US or Commonwealth spelling.